web/index.html

```diff
@@ -2,20 +2,14 @@
 <html ng-app="konnektApp">
   <head>
     <script type="text/javascript" src="dist/bundle.js"></script>
-
     <meta charset="utf-8">
     <title>Konnekt Kontakt Manager</title>
-
     <link href="https://fonts.googleapis.com/css?family=Merriweather:400i" rel="stylesheet">
     <link href="https://fonts.googleapis.com/css?family=Raleway:800" rel="stylesheet">
-
-
   </head>
   <body>
-    <img class="logo-feher-hosszu" src="/dist/images/logo-feher-hosszu.png" alt="Konnekt Kontakt Manager">
-    <div class="rectangle" ng-view>
-      <!-- registration, login code comes here -->
-
-    </div>
+    <div class="konnekt-logo"></div>
+    <div class="spacer"></div>
```

Comment:
if you want to have space below the konnekt-logo, add some margin-bottom to it instead